### PR TITLE
iOS-3366 Reactions | show profile icon for one reaction (to branch)

### PR DESCRIPTION
**Pull request by mgolovko**

### CODE REVIEW

File: Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift [hunk 1]
```diff
@@ -14,10 +14,14 @@ final class DiscussionViewModel: ObservableObject, MessageModuleOutput {
...
     @Injected(\.chatService)
     private var chatService: any ChatServiceProtocol
+    @Injected(\.accountParticipantsStorage)
+    private var accountParticipantsStorage: any AccountParticipantsStorageProtocol
     
     private lazy var participantSubscription: any ParticipantsSubscriptionProtocol = Container.shared.participantSubscription(spaceId)
     private lazy var chatStorage: any ChatMessagesStorageProtocol = Container.shared.chatMessageStorage(chatId)
     
+    
+
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift:24]: looks like redundant spaces)

File: Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift [hunk 2]
```diff
@@ -104,13 +108,36 @@ final class DiscussionViewModel: ObservableObject, MessageModuleOutput {
     
     private func updateMessages() {
         
-        let newMesageBlocks = messages.compactMap { message  in
-            MessageViewData(
+        let yourProfileIdentity = accountParticipantsStorage.participants.first?.identity
+        
+        let newMesageBlocks = messages.compactMap { message -> MessageViewData in
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift:113]: newMesageBlocks -> newMessageBlocks

**Review by joe-pusya — APPROVED** [review]

File: Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift (hunk 1 above)
Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift:24]: fixed!

File: Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift (hunk 2 above)
Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/Modules/Discussion/DiscussionViewModel.swift:113]: fixed

**Pull request merged**